examples/Lorenz63/Lorenz63.jl

```diff
@@ -0,0 +1,188 @@
+using JLD2
```

Comment:
JLD might still be a better idea than JLD2. See from the JLD2 GitHub: _Please use caution. If your tolerance for data loss is low, JLD may be a better choice at this time._

It also offers more compatibility with the HDF5 format, so we can read results in Python for example.

Reply:
Initially I had no intentions to set a precedent with this. So long as we can store the objects we desire then I don't mind which we use.  Though this is consistent with the current codebase - so perhaps raise an issue and this can be resolved in another PR.